scripts/kfctl.sh

```diff
@@ -15,9 +15,14 @@ WHAT=$2
 
 ENV_FILE="env.sh"
 
+KUBEFLOW_VERSION=${KUBEFLOW_VERSION:-"master"}
+KUBEFLOW_DEPLOY=${KUBEFLOW_DEPLOY:-true}
```

Comment:
We shouldn't need KUBEFLOW_DEPLOY anymore

Reply:
removed